### PR TITLE
a set of generators & test cases for major POIs generated from wikipedia

### PR DESCRIPTION
**Pull request by blackmad**
This is partly here as a PoC

I'm curious if you'd want me to integrate it into the scripts in fuzzy-tester. It seems like it's different enough (doesn't use csvs as input) and relies on a bunch of libraries that I'm not sure if it should be integrated into that existing flow. I'm happy to make it work better with that though.

I was curious to see how we're doing on standalone major POIs so ... I built this. It mostly works. wikipedia is a pain.

### CODE REVIEW

**Comment by orangejulius** [comment]
Once again, these are awesome.

I thought of the existing scripts in `fuzzy-tester` too, and honestly, this is a better place. Eventually we should move the existing scripts here.

File: test_cases/List_of_U.S._stadiums_by_capacity.json [hunk 1]
```diff
@@ -0,0 +1,3310 @@
...
+        "coordinates": [
+          -96.34028,
+          30.61
+        ],
+        "distanceThresh": 5000,
+        "properties": {
+          "layer": "venue"
+        }
+      }
+    },
```
Comment (orangejulius) [inline test_cases/List_of_U.S._stadiums_by_capacity.json]: We will probably want an expected `name` field. Right now all these tests will pass as long as any venue within 5km of the given point is returned. While that's nice for getting around minor name differences it's probably too lenient :(

**Comment by blackmad** [comment]
poke

**Comment by blackmad** [comment]
lmk if you want to land this?

**Comment by blackmad** [comment]
@orangejulius lmk if you want to land this or I can close. Trying to clean up my backlog of open / unfinished tasks

**Pull request closed without merging**